src/Examples/UnitModels/Operations/product_testing.ipynb

```diff
@@ -0,0 +1,291 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Tutorial: Product Unit Model with Modular Property Package\n",
+    "\n",
+    "![](product.svg)\n",
+    "\n",
+    "## Learning Outcomes\n",
+    "\n",
+    "- Demonstrate use of the Product unit model in IDAES\n",
+    "\n",
+    "\n",
+    "## Problem Statement\n",
+    "\n",
+    "In this tutorial, we will be feeding a mixed stream of liquid benzene and liquid toluene. This unit model has no outlets and only one inlet. The inlet conditions are as follows:\n",
```

Comment:
Again, an explanation of why you want to include this in a flowsheet is probably required - it is mostly for reporting and visualization purposes as you can build and run a flowsheet without a produce block if you desire.